clarifai/cli/compute_cluster.py

```diff
@@ -0,0 +1,47 @@
+import click
+from clarifai.cli.base import cli
+from clarifai.client.user import User
+from clarifai.utils.cli import display_co_resources
+
+
+@cli.group()
+@click.pass_context
+def computecluster(ctx):
+  """Manage Compute Clusters: create, delete, list"""
+  ctx.obj = User(user_id=ctx.obj['user_id'], pat=ctx.obj['pat'], base_url=ctx.obj['base_url'])
+
+
+@computecluster.command()
+@click.option(
+    '--compute_cluster_id',
+    required=True,
+    help='Compute Cluster ID for the compute cluster to create.')
+@click.option(
+    '--config_filepath',
```

Comment:
We can also add an alias `config` here, WDYT?

Reply:
Yeah, adding config `alias` would be better

Reply:
Done